pages/k8s/how-to-install.md

```diff
@@ -0,0 +1,38 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Installing Charmed Kubernetes"
+  description: installing to different substrates
+keywords: install, , metallb, loadbalancer
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: how-to-install.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+In addition to the easy to follow [tutorial](/kubernetes/docs/quickstart) for
+Charmed Kubernetes, additional guides are available to take you through the
+installation steps for a number of different substrates. The 'Cloud' install
+page covers many different scenarios with Juju-supported clouds.
+
+- [Install on a cloud](/kubernetes/docs/install-manual)
+- [Install locally with LXD](/kubernetes/docs/install-local)
+- [Install on Equinix](/kubernetes/docs/equinix)
+
+There are also two 'special case' scenarios we provide guidance for:
+
+- [Installing offline, or in a restricted environment ](/kubernetes/docs/install-offline)
```

Comment:
I fear this extra space is going to break some ridiculous tooling somewhere down the line.
```suggestion
- [Installing offline, or in a restricted environment](/kubernetes/docs/install-offline)
```